<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="RadiusLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="AreaLabelText">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="AreaLabel">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="RadiusLineLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CountButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <zorder>AreaLabelText</zorder>
  <zorder>AreaLabel</zorder>
  <zorder>RadiusLineLabel</zorder>
  <zorder>CountButton</zorder>
  <zorder>RadiusLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
